fili-core/src/main/java/com/yahoo/bard/webservice/data/config/table/BaseTableLoader.java

```diff
@@ -159,6 +167,138 @@ public void loadLogicalTableWithGranularities(
     }
 
     /**
+     * Iterate through given definitions and builds all corresponding physical table with satisfied dependency.
+     *
+     * @param currentTableName  Iterator for the mutable definition
+     * @param availableTableDefinitions  A map of table name to table definition that are available globally
```

Comment:
`availableTableDefinitions` simply means the table definitions that is available for you to use, it may hold definitions that is already built, so better not use untouched or unbuilt